doc/COMPILING/COMPILING-VS-VCPKG.md

```diff
@@ -81,16 +85,26 @@ Even if you do not need languages other than English, you may still want to exec
 2. Configure the working directory in the project settings to `$(ProjectDir)..`
 
     - Right click the project in the Solution Explorer pane, select `Properties`
+    - Select Configuration to `All Configurations` and Platform to `All Platforms`
     - Under `Configuration Properties > Debugging`, change `Working Directory` to `$(ProjectDir)..`
-    - **Note:** If you change configurations (eg from `Release` to `Debug`), you will have to repeat this step. You only have to do it once per configuration, the setting will persist.
 
 If you discover that after pressing the debug button in Visual Studio, Cataclysm just exits after launch with return code 1, that is because of the wrong working directory.
 
 When debugging, it is not strictly necessary to use a `Debug` build; `Release` builds run significantly faster, can still be run in the debugger, and most of the time will have most of the information you need.
 
 ### Running unit tests
 
-Ensure that the Cataclysm test project (`Cataclysm-test-vcpkg-static`) is the selected startup project, configure the working directory in the project settings to `$(ProjectDir)..`, and then press the debug button (or use the appropriate shortcut, e.g. F5). This will run all of the unit tests.
+1. Ensure that the Cataclysm test project (`Cataclysm-test-vcpkg-static`) is the selected startup project.
+
+2. Configure the working directory in the project settings to `$(ProjectDir)..`
+
+3. Configure the command line arguments for the tests
+
+    - Under `Configuration Properties > Debugging`, change `Command Arguments` to `--wait-for-keypress exit`
+
+4. Press the debug button (or use the appropriate shortcut, e.g. F5). This will run all of the unit tests.
+
+    - The windows console will stay open until you press enter. This because of `--wait-for-keypress exit`.
```

Comment:
```suggestion
    - The windows console will stay open until you press Enter. This is because of `--wait-for-keypress exit`.
```

Reply:
Nitpick: It should be "Windows console" instead of "windows console"